.github/workflows/build-cli.yml

```
@@ -138,11 +143,9 @@ jobs:
           ResourceHacker -open version-info.rc -save version-info.res -action compile
           ResourceHacker -open %WIN_PKG_BUILT% -save %WIN_PKG_BUILT% -action addoverwrite -resource version-info.res
 
-      - name: Setup sub-module
-        run: npm run sub:init
-
       - name: Install
         run: npm ci
+        working-directory: ./
```

I would guess this is what's causing the yaml lint failures, I think this needs to be wrapped in quotes
```suggestion
        working-directory: './'
```

---

The builds have been broken for awhile, and I don't think it,s just this file.

---

The yaml linter is failing because of a "too many requests" from GitHub when we check the latest versions of the pinned actions. It is currently being worked on for a fix